tests/time_tests/c++/pipeline.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "timer.h"
+
+#include <inference_engine.hpp>
+using namespace InferenceEngine;
+
+
+int runPipeline(const std::string &model, const std::string &device) {
+    start_timer("first_time_inference");
+
+    Core ie;
+
+    start_timer("read_network");
+    CNNNetwork cnnNetwork = ie.ReadNetwork(model);
+    finish_timer("read_network");
```

Comment:
We should not remember the timer name. Scoped measure would help here.

Reply:
In this case we still need to handle timer name, but scoped measure doesn't require name specification more than once

Reply:
Fixed